web/src/components/molecules/Asset/Asset/AssetBody/no-supported-viewer.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from "@emotion/styled";
+import { twoTone } from "@reearth-cms/components/atoms/Icon";
+
+const NoSupportedViewer: React.FC = () => {
```

Comment:
This is being used as a fallback for if the asset type isn't found/supported, right? Is it that a "viewer" is not supported? Or file format?
If it IS viewer, `ViewerNotSupported` might be clearer.
If it isn't, perhaps `FormatNotSupported` or `AssetNotSupported` or something similar maybe is better?

Also below `Didn't find supported viewer` -> `Not supported` or `Unsupported format` or something similar.

Reply:
I am naming according to the figma design.

<img width="1214" alt="Screen Shot 0004-07-28 at 5 43 13 PM" src="https://user-images.githubusercontent.com/35835415/181463944-418dd9e7-1387-4e46-b530-95b83b5a020f.png">

that is why I use "Viewer", if the design file is correct then I think that I will rename to "ViewerNotSupported"

Reply:
Renamed to ViewerNotSupported

Reply:
This is a difficult point in our dev process.. Since I am the only native English speaker language will not always be correct or appropriate and I don't have enough time to spell-check everyone's English. I feel like you are very comfortable with English so if you ever find something in design files that seems a little wrong or you have a better way to say it definitely ask @lavalse and if he doesn't know you guys can quickly ask me.

Reply:
As for the wording in this case, I just talked with @lavalse and I think everything is fine here now that you updated ViewerNotSupported!

Reply:
Ok, I didn't know that. I thought that I should write what is written in the design files.
From now on, if I ever find something in design files that seems a little wrong, I will discuss it with you and red.